{{cookiecutter.__src_folder_name}}/infra/core/host/container-app.bicep

```diff
@@ -130,7 +130,9 @@ resource app 'Microsoft.App/containerApps@2023-05-02-preview' = {
       } : { enabled: false }
       secrets: [for secret in items(secrets): {
```

Comment:
This is different to the base templates, I can't see another way unless there's a method of not doing keyVaultUrl

Reply:
Looks good as long as its okay with the null values!